app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.errors import VaccineError, NotWearingMaskError
```

Comment:
it is a good practice to arrange imports alphabetically

Reply:
It was mentioned in the lectures and it was said that it is not mandatory. I think this practice will not improve readability.